packages/schema/lib/schemas/BasicOperationSchema.js

```diff
@@ -50,6 +50,33 @@ module.exports = makeSchema(
           },
         },
       },
+      lock: {
+        description:
+          '**INTERNAL USE ONLY**. Zapier uses this config for internal operation locking.',
+        type: 'object',
+        required: ['key'],
+        properties: {
+          key: {
+            description:
+              'The key to use for locking. This should be unique to the operation.',
+            type: 'string',
+            minLength: 1,
+          },
+          scope: {
+            description: `The level at which an app's access is restricted to. By default, locks are scoped to the app. That is, all users of the app will share the same locks. If you want to restrict serial access to a specific user, auth, or account, you can set the scope to one or more of the following: 'user' - Locks based on user ids.  'auth' - Locks based on unique auth ids. 'account' - Locks for all users under a single account. You may also combine scopes.`,
```

Comment:
I find "The level at which an app's access is restricted to." and the term "access" confusing, but what follows very clear, so I suggest we drop that first sentence.

I also suspect/expect that `app` would always be included as scope, otherwise one integration could block parallel execution of another that happens to use the same scope and e.g. `account` scope. That isn't clear now.